console/views.py

```diff
@@ -1,15 +1,19 @@
-from flask import request, render_template, jsonify
-
-from console import app
-
 import console.settings as settings
-from structlog import wrap_logger
 
-import json
 import logging
 import logging.handlers
-import time
+from structlog import wrap_logger
+
+from threading import Timer
 
 
 logging.basicConfig(level=settings.LOGGING_LEVEL, format=settings.LOGGING_FORMAT)
 logger = wrap_logger(logging.getLogger(__name__))
+
+
+def heartbeat(logger=logger, heartbeats=0):
+    logger.info("Heartbeat " + str(heartbeats))
```

Comment:
You've got a `structlog` so put the heartbeats into a key/value pair:
```python
logger.info("Heartbeat", heartbeats=heartbeats)
```

Though do we need the actual number of beats? What's the maximum storable int value in `heartbeats` and what would happen on overflow if we just keep counting up (and the service stays up).

Is there a performance/memory implication for the recursive calls?